examples/model-comparison.py

```diff
@@ -31,8 +31,8 @@
 import numpy as np
 
 # Plot integrated power and capacity
-T_out = np.arange(-23,75+1,1)
-conditions = [t24_unit.make_condition(resdx.HeatingConditions,outdoor=resdx.PsychState(drybulb=resdx.fr_u(T,"°F"),wetbulb=resdx.fr_u(T-2.0,"°F"))) for T in T_out]
+T_out = np.arange(35,75+1,1)
```

Comment:
Why change the temperature range here?

Reply:
Conditions are not real. see response above.